package.json

```diff
@@ -47,7 +47,7 @@
     "@types/jest-plugin-context": "^2.9.2",
     "@types/js-yaml": "^4.0.9",
     "@types/lambda-tester": "^4.0.3",
-    "@types/node": "^20.12.7",
+    "@types/node": "^22.7.5",
```

Comment:
Any reason for this instead of staying at v20.x?

Reply:
Yeah agree. I tried jumping it down to 18 first but the only way I could fix the package version issues I was having was to bump this to 22. My thoughts basically came down to I think it is probably fine to be ahead on these type declerations, but if an issue I can go back to the drawing board on getting a version of this that actually works.

Reply:
Just installed `"@types/node": "^18.19.55",` and performed a `npm i` & `npm ci` - they seem ok for me, tests still passing via `npm run test` etc.

Whereabouts were you seeing the package version issue you mentioned?